CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -585,8 +588,18 @@ protected virtual bool TryGetProperty(long objectId, string name, out object res
 
             try
             {
+#if NETCOREAPP
                 result = property.GetValue(obj.Value);
```

Comment:
We should be able to exclude the whole method for simplicity.

Reply:
@amaitland Excluded the TryGetPropperty for .Net core App.